typespec-ccf/main.tsp

```diff
@@ -1,608 +1,22 @@
-    @body tlsCaCertBundles: string;
-};
-
-@route("/gov/kv/users/certs")
-@get op getUsersCerts(): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body usersCerts: UserIdToPem;
-};
-
-@route("/gov/kv/users/info")
-@get op getUsersInfo(): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body usersInfo: UserDetails;
-};
-
-@route("/gov/proposals")
-namespace proposal {
-    @summary("Proposed changes to the service pending resolution")
-    @get op allProposals(): {
-        @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body proposals: StringToProposalInfo
-    };
-    @summary("Submit a proposed change to the service")
-    @post op proposalInfo(@body proposal: Proposal): {
-        @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body proposalInfoDetail: ProposalInfoSummary;
-    };
-}
-
-@route("/gov/proposals/{proposal_id}")
-@summary("Information about a proposed change to the service")
-@get op getProposalId(@path proposal_id: string): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body proposalInfoDetail: ProposalInfo;
-};
-
-@route("/gov/proposals/{proposal_id}/actions")
-@summary("Actions contained in a proposed change to the service")
-@get op getProposalIdActions(@path proposal_id: string): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body proposalActions: Proposal;
-};
-
-@route("/gov/proposals/{proposal_id}/ballots")
-@summary("Ballots submitted against a proposed change to the service")
-@post op postProposalIdBallots(@path proposal_id: string, @body ballot: Ballot): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body ballotInfo: ProposalInfoSummary;
-};
-
-@route("/gov/proposals/{proposal_id}/ballots/{member_id}")
-@summary("Ballot for a given member about a proposed change to the service")
-@get op getMemberIdBallot(@path proposal_id: string, @path member_id: string): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body ballot: Ballot;
-};
-
-@route("/gov/proposals/{proposal_id}/withdraw")
-@summary("Withdraw a proposed change to the service")
-@post op postProposalIdWithdraw(@path proposal_id: string): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body proposalInfoDetail: ProposalInfo;
-};
-
-@route("/gov/receipt")
-@doc("A signed statement from the service over a transaction entry in the ledger")
-@summary("Receipt for a transaction")
-@get op getReceipt(@query @pattern("^[0-9]+\\.[0-9]+$")transactionID: string): {
-    @body receipt: Json;
-};
-
-@route("/gov/recovery_share")
-@summary("Provide a recovery share for the purpose of completing a service recovery")
-@post op postRecoveryShare(@body recoveryShare: SubmitRecoveryShareIn): {
-    @header("x-ms-ccf-transaction-id") transactionID: string;
-    @body recoveryShare: SubmitRecoveryShareOut;
-};
-
-@route("/gov/encrypted_recovery_share/{member_id}")
```

Comment:
I know it's in-progress so this is probably coming, but just noting that all the recovery-related endpoints are gone now